project/ws/app/src/lib/routes/app-toc/components/app-toc-banner/app-toc-banner.component.ts

```diff
@@ -287,8 +287,11 @@ export class AppTocBannerComponent implements OnInit, OnChanges, OnDestroy {
     })
   }
 
-  downloadPdf(content: any) {
-    return content
+  downloadCertificate(content: any) {
+    this.contentSvc.downloadCertificateAPI(content.identifier).toPromise().then((data: any) => {
+      // tslint:disable-next-line:no-console
+      console.log(data)
```

Comment:
remove console.log and run lint fix

Reply:
Yes, I know. I had added it, because the content progress API was not working then and hence had added  the tslint  
// tslint:disable-next-line:no-console
Every time the code is committed and pushed, tslint and npm run build always get's executed